package.json

```diff
@@ -285,9 +285,8 @@
 		"test-server:watch": "npm run -s test-server -- --watch",
 		"translate": "i18n-calypso --format pot --output-file ./calypso-strings.pot -k translate,__,_x,_n,_nx -e date '**/*.js' '**/*.jsx' '**/*.ts' '**/*.tsx' '!build/**' '!packages/**/dist/**' '!public/**' '!**/test/**' '!node_modules/**' '!apps/full-site-editing/**' '!**/node_modules/**'",
 		"typecheck": "tsc --noEmit",
-		"update-deps": "npx rimraf npm-shrinkwrap.json && npm run -s distclean && npm i && npm shrinkwrap && replace --silent 'http://' 'https://' . --recursive --include='npm-shrinkwrap.json,package-lock.json'",
+		"update-deps": "npx rimraf package-lock.json && npm run -s distclean && npm i && replace --silent 'http://' 'https://' . --recursive --include='package-lock.json'",
 		"validate-fallback-es5": "npx eslint --env=es5 --no-eslintrc --no-ignore ./public/fallback/*.js",
-		"postshrinkwrap": "node -e \"fs.utimesSync( './node_modules', new Date(), new Date() );\"",
```

Comment:
Rather than removing this, should it be `postupdate-deps`?

It looks like this may be to prevent us from triggering `install-if-deps-outdated` after `update-deps`. Would we lose this with this change or is this redundant?

Reply:
That's right, we need to ensure that after `npm run update-deps`, the `node_modules` folder is newer than `package-lock.json` file. Without the `postshrinkwrap` code, the `package-lock.json` file is some 10-15 seconds newer. The `install-if-deps-outdated` script would reinstall `node_modules` without it being needed.

I added a `touch` call to the end of the `update-deps` script in https://github.com/Automattic/wp-calypso/pull/37493/commits/18376ed0a6de3dcdb435a79917b7bd398c480858

I'm not sure if using `touch` (Unix utility) is OK, or if doing that in pure Node.js would be preferred.